parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -71,12 +76,24 @@ installNvidiaFabricManager() {
 installNvidiaContainerRuntime() {
     MARINER_NVIDIA_CONTAINER_RUNTIME_VERSION="3.13.0"
     MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION="1.13.5"
+
+    if [[ $OS_VERSION == "3.0" ]]; then
```

Comment:
[nit] the other conditions are consistent in checking for 2.0 instead of 3.0

Reply:
Made a change for this - [commit](https://github.com/Azure/AgentBaker/pull/4808/commits/8bac55a3ebee4d9bca3e8b306532a9bc4ee92ebb)